out/latest-types

```diff
@@ -0,0 +1,257 @@
+Types minmax version:
```

Comment:
that is really amazing! If I could make a wish a would love to have a CI job that generates all types for all runtimes like that! 😍

Reply:
Not sure what do you mean in the last point. Anyway, might be a good task for another PR to define some CI jobs, even thought manual types should be outdated soon...

Reply:
Implemented a basic CI action to generate new types when merged into `master`. Tested locally on my branch and got it working, so I expect it to work also when pushed on master.